_docs/reference/release-notes.md

```diff
@@ -8,91 +8,34 @@ layout: docs
-- **Ingress Policies**. In addition to east-west traffic supported in 0.1. policies can now be applied to north-south traffic.
-
-- **Support for TCP Services**. In addition to the HTTP-level policy controls available in 0.1, 0.2 introduces policy controls for 
-TCP services.
-
-- **New Mixer API**. The API that Envoy uses to interact with Mixer has been completely redesigned for increased robustness, flexibility, and to support 
-rich proxy-side caching and batching for increased performance.
-
-- **New Mixer Adapter Model**. A new adapter composition model makes it easier to extend Mixer by adding whole new classes of adapters via templates. This 
-new model will serve as the foundational building block for many features in the future. See the
-[Adapter Developer's Guide](https://github.com/istio/istio/blob/master/mixer/doc/adapters.md) to learn how
-to write adapters.
-
-- **Improved Mixer Build Model**. It’s now easier to build a Mixer binary that includes custom adapters.
-
-- **Mixer Adapter Updates**. The built-in adapters have all been rewritten to fit into the new adapter model. The stackdriver adapter has been added for this
-release. The experimental redis quota adapter has been removed in the 0.2 release, but is expected to come back in production quality for the 0.3 release.
-
-- **Mixer Call Tracing**. Calls between Envoy and Mixer can now be traced and analyzed in the Zipkin dashboard.
-
-#### Security
-
-- **Mutual TLS for TCP Traffic**. In addition to HTTP traffic, mutual TLS is now supported for TCP traffic as well.
-
-- **Identity Provisioning for VMs and Physical Machines**. Auth supports a new mechanism using a per-node agent for
-identity provisioning. This agent runs on each node (VM / physical machine) and is responsible for generating and sending out the CSR
-(Certificate Signing Request) to get certificates from Istio CA.
-
-- **Bring Your Own CA Certificates**. Allows users to provide their own key and certificate for Istio CA.
-
-- **Persistent CA Key/Certificate Storage**. Istio CA now stores signing key/certificates in
-persistent storage to facilitate CA restarts. 
-
-### Known Issues
-
-- **User may get periodical 404 when accessing the application**:  We have noticed that Envoy doesn't get routes properly occasionally thus a 404 is returned to the user.  We are actively working on this [issue](https://github.com/istio/istio/issues/1038).
-- **Istio Ingress or Egress reports ready before Pilot is actually ready**: You can check the istio-ingress and istio-egress pods status in the `istio-system` namespace and wait a few seconds after all the Istio pods reach ready status.  We are actively working on this [issue](https://github.com/istio/istio/pull/1055).
-- **A service with Istio Auth enabled can't communicate with a service without Istio**: This limitation will be removed in the near future.
+- **Selective Auth**. You can now control authentication on a per-service basis via service annotations. [TBD: DOC LINK?]
```

Comment:
that's one of the major features in this release
https://istio.io/faq/#collapse13
(the link doesn't actually work, but... click "security" then click "Can I enable istio auth with some services...")

cc @wattli

Reply:
Can we get a real documentation link? The FAQ can't be the only place we explain how to do this...

Reply:
@diemtvu to help provide the doc link to per-service auth enablement